apps/sms/style/sms.css

```diff
@@ -464,7 +463,7 @@ section[role="region"].new > header:first-child form {
   color: #008787;
 }
 
-.has-carrier .carrier-wrapper {
+.has-carrier {
```

Comment:
and this is the rule that overrides the "display: none" above. So you need to keep it as well, I think ?